relayer/workers/parachaincommitmentrelayer/beefy-listener.go

```diff
@@ -269,96 +260,15 @@ func (li *BeefyListener) GetAllParaheads(blockHash types.Hash, ourParachainId ui
-		}
-		allParaHeadsBytes = append(allParaHeadsBytes, paraHeadBytes)
-	}
-	ourParaHeadBytes, err := types.EncodeToBytes(ourParaHead)
-	if err != nil {
-		return [][32]byte{}, err
-	}
-
-	paraTreeData := make([][]byte, len(allParaHeadsBytes))
-	for i, paraHead := range allParaHeadsBytes {
-		paraTreeData[i] = paraHead
-	}
-
-	// Create the tree
-	paraMerkleTree, err := merkletree.NewUsing(paraTreeData, &Keccak256{}, nil)
-	if err != nil {
-		return [][32]byte{}, err
-	}
-
-	// Generate Merkle Proof for our parachain's head
-	proof, err := paraMerkleTree.GenerateProof(ourParaHeadBytes)
-	if err != nil {
-		return [][32]byte{}, err
-	}
-
-	// Verify the proof
-	root := paraMerkleTree.Root()
-	verified, err := merkletree.VerifyProofUsing(ourParaHeadBytes, proof, root, &Keccak256{}, nil)
-	if err != nil {
-		return [][32]byte{}, err
-	}
-	if !verified {
-		return [][32]byte{}, fmt.Errorf("failed to verify proof")
-	}
-
-	proofContents := make([][32]byte, len(proof.Hashes))
-	for i, hash := range proof.Hashes {
-		var hash32Byte [32]byte
-		copy(hash32Byte[:], hash)
-		proofContents[i] = hash32Byte
-	}
-
-	fmt.Println("parachain-commitment-relayer allParaHeadsBytes", allParaHeadsBytes)
-	allParaHeadsBytesHex, _ := types.EncodeToHexString(allParaHeadsBytes)
-	fmt.Println("parachain-commitment-relayer allParaHeadsBytesHex", allParaHeadsBytesHex)
-
-	paraHeadBytes0Hex, _ := types.EncodeToHexString(allParaHeadsBytes[0])
-	fmt.Println("parachain-commitment-relayer paraHeadBytes0Hex", paraHeadBytes0Hex)
-	paraHeadBytes1Hex, _ := types.EncodeToHexString(allParaHeadsBytes[1])
-	fmt.Println("parachain-commitment-relayer paraHeadBytes1Hex", paraHeadBytes1Hex)
-	fmt.Println("parachain-commitment-relayer paraHeadBytesHex", paraHeadBytes0Hex, paraHeadBytes1Hex)
-
-	fmt.Println("parachain-commitment-relayer ourParaHeadBytes", ourParaHeadBytes)
-	ourParaHeadBytesHex, _ := types.EncodeToHexString(ourParaHeadBytes)
-	fmt.Println("parachain-commitment-relayer ourParaHeadBytesHex", ourParaHeadBytesHex)
-	rootHex, _ := types.EncodeToHexString(root)
-	fmt.Println("parachain-commitment-relayer root", rootHex)
-	fmt.Println("parachain-commitment-relayer proof", proof)
-	fmt.Println("parachain-commitment-relayer len(proof.Hashes)", len(proof.Hashes))
-	fmt.Println("parachain-commitment-relayer proofContents", proofContents)
-	proofContents0Hex, _ := types.EncodeToHexString(proofContents[0])
-	fmt.Println("parachain-commitment-relayer proofContents0Hex", proofContents0Hex)
-
-	return proofContents, nil
-}
-
-// Keccak256 is the Keccak256 hashing method
-type Keccak256 struct{}
-
-// New creates a new Keccak256 hashing method
-func New() *Keccak256 {
-	return &Keccak256{}
-}
-
-// Hash generates a Keccak256 hash from a byte array
-func (h *Keccak256) Hash(data []byte) []byte {
-	hash := crypto.Keccak256(data)
-	return hash[:]
+func createParachainHeaderProof(allParaHeads []types.Header, ourParaHead types.Header) string {
+	//TODO: implement
+	return ""
```

Comment:
Is the code above you removed no longer applicable?

Reply:
it is in another open PR, so I was planning to merge this one to main and let it continue in that PR